.github/workflows/build-apko-image.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and upload all apko container images
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - "release/**"
+      - "feat/reproducible-builds-ko-nogcp"
```

Comment:
before merging, same for other workflow files
```suggestion
```

Reply:
Yes, I added this branch filter to run the tests on this branch, this will be removed before merge.